data/core/terrain-graphics.cfg

```diff
@@ -519,7 +539,7 @@ C*,K*,X*,Q*,W*,Ai,M*,*^V*,*^B*,_off^_usr#enddef
 {OVERLAY_P               *^Edp,*^Edpp                                    32        embellishments/desert-plant9}
 {OVERLAY_P               *^Edp,*^Edpp                                    48        embellishments/desert-plant10}
 {OVERLAY_P               *^Edp,*^Edpp                                    100       embellishments/desert-plant11}
-
+#endif
```

Comment:
What's this doing here? I can't find a matching `#if` in the diff, nor an `#endif` that was removed elsewhere.

Reply:
There's an `#ifdef __UNUSED__ ` a few lines up, it's a bulk comment-out

Reply:
Ohh, I see it now. Not sure how I missed that.